ReadMe.md

```diff
@@ -39,7 +39,16 @@ sudo apt-get install oce-draw liboce-modeling-dev liboce-ocaf-dev
 
 Building occQt on Windows
 =========================
+
+For QtCreator
+-------------------------------
+Open the occQt.pro from the Qt menu and set 
+the  variable CASROOT  Opencascade installation path
+
+For VisualStudio 2017
```

Comment:
I think change For VisualStudio 2017 to For VisualStudio is better, because occQt can be compiled with 2013, 2015, .etc.